cmd/utils.go

```diff
@@ -445,54 +461,69 @@ func getConfigLabels(config *RootCommandConfig) (map[string]string, error) {
 		return labels, projectConfigErr
 	}
 
-	labels["appsody.requested-stack"] = projectConfig.Stack
+	t := time.Now()
 
-	return labels, nil
-}
+	labels[ociKeyPrefix+"created"] = t.Format(time.RFC3339)
 
-func getGitLabels(config *RootCommandConfig) (map[string]string, error) {
-	gitInfo, err := GetGitInfo(config.Dryrun)
-	if err != nil {
-		return nil, err
+	var maintainersString string
+	for index, maintainer := range projectConfig.Maintainers {
+		maintainersString += maintainer.Name + ", " + maintainer.Email + ", " + maintainer.GithubID
+		if index < len(projectConfig.Maintainers)-1 {
+			maintainersString += "-"
+		}
 	}
 
-	var labels = make(map[string]string)
+	if maintainersString != "" {
+		labels[ociKeyPrefix+"authors"] = maintainersString
+	}
 
-	gitString := "git."
-	if gitInfo.Branch != "" {
-		labels[gitString+"branch"] = gitInfo.Branch
+	if projectConfig.Version != "" {
+		labels[ociKeyPrefix+"version"] = projectConfig.Version
 	}
 
-	if gitInfo.Upstream != "" {
-		labels[gitString+"upstream"] = gitInfo.Upstream
+	if projectConfig.License != "" {
+		labels[ociKeyPrefix+"licenses"] = projectConfig.License
 	}
 
-	if gitInfo.RemoteURL != "" {
-		labels[gitString+"remoteurl"] = gitInfo.RemoteURL
+	if projectConfig.ProjectName != "" {
+		labels[ociKeyPrefix+"title"] = projectConfig.ProjectName
+	}
+	if projectConfig.Description != "" {
+		labels[ociKeyPrefix+"description"] = projectConfig.Description
 	}
 
-	if gitInfo.ChangesMade {
-		labels[gitString+"changesmade"] = "true"
-	} else {
-		labels[gitString+"changesmade"] = "false"
+	if projectConfig.Stack != "" {
+		labels[appsodyKeyPrefix+"configured"] = projectConfig.Stack
 	}
 
-	var commitInfo = gitInfo.Commit
-	commitString := "git.commit."
-	if commitInfo.Author != "" {
-		labels[commitString+"author"] = commitInfo.Author
+	if projectConfig.ApplicationName != "" {
+		labels["dev.appsody.application"] = projectConfig.ApplicationName
 	}
 
-	if commitInfo.SHA != "" {
-		labels[commitString+"sha"] = commitInfo.SHA
+	return labels, nil
+}
+
+func getGitLabels(config *RootCommandConfig) (map[string]string, error) {
+	gitInfo, err := GetGitInfo(config.Dryrun)
+	if err != nil {
+		return nil, err
 	}
 
-	if commitInfo.Date != "" {
-		labels[commitString+"date"] = commitInfo.Date
+	var labels = make(map[string]string)
+
+	if gitInfo.RemoteURL != "" {
+		labels[ociKeyPrefix+"url"] = gitInfo.RemoteURL
+		labels[ociKeyPrefix+"documentation"] = gitInfo.RemoteURL
+		labels[ociKeyPrefix+"source"] = gitInfo.RemoteURL + "tree/" + gitInfo.Branch
 	}
 
-	if commitInfo.URL != "" {
-		labels[commitString+"url"] = commitInfo.URL
+	var commitInfo = gitInfo.Commit
+	revisionKey := appsodyKeyPrefix + "revision"
+	if commitInfo.SHA != "" {
+		labels[revisionKey] = commitInfo.SHA
+		if gitInfo.ChangesMade {
+			labels[revisionKey] += "-changesMade:true"
```

Comment:
I'm not necessarily opposed to this, but I'm wondering your reason for using "-changesMade:true" instead of "-modified" as discussed?